.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,160 @@
+name: Evmone Coverage Report
+
+on:
+  pull_request:
+    paths:
+      - 'converted-ethereum-tests.txt' # This triggers the workflow only for changes in file.txt
+
+jobs:
+  evmone-coverage-diff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache EVMONE
+        uses: actions/cache@v2
+        with:
+          path: ${{ github.workspace }}/evmoneimage
+          key: ${{ runner.os }}-evmone-${{ hashFiles('evmone-hash.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-evmone-
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }} # Checks out the PR branch
+          fetch-depth: 0 # Necessary to fetch all history for diff
+
+      - name: Fetch target branch
+        run: git fetch origin ${{ github.base_ref }}:refs/remotes/origin/${{ github.base_ref }}
+
+      - name: Install deps
+        run: |
+          echo $(pwd)
+          echo ${{ github.workspace }}
+
+          sudo apt-get install docker-ce
+
+          #install pyspec deps from root repo
+          python3 --version
+          python3 -m venv ./venv/
+          source ./venv/bin/activate
+          pip install -e .
+
+
+          #install go
+          curl -L --output go1.21.8.linux-amd64.tar.gz "https://go.dev/dl/go1.21.8.linux-amd64.tar.gz"
+          tar -xf go1.21.8.linux-amd64.tar.gz
+          sudo mv go /usr/local
+          sudo ln -s /usr/local/go/bin/go /usr/local/bin/go
+          git clone --depth 1 https://github.com/ethereum/go-ethereum.git
+          cd go-ethereum
+          go build ./cmd/evm
+          cp evm /usr/local/bin
+          cd ..
```

Comment:
We have an action to do this more easily:
```yaml
- name: Build EVM
        uses: ./.github/actions/build-evm
        id: evm-builder
        with:
          type: 'main'
```

Reply:
does it also install solc?